### PR TITLE
Generate environments based on loaded configration

### PR DESCRIPTION
**Pull request by bregman-arie**
Once the app has loaded the configuration, it has all
the information required for creating the different
Environment instances.

These environments will be later populated with information
gathered from the different sources.

### CODE REVIEW

**Comment by bregman-arie** [comment]
@jsanemet this part mainly depends on your implementation of the Config component so I'll wait until it's merged to continue the work on it.

**Review by amolkahat — APPROVED** [review]

**Review by adrianfusco — APPROVED** [review]

File: cibyl/models/ci/system.py
Comment (cescgina) [inline cibyl/models/ci/system.py:50]: Shouldn't this be AttributeListValue?

**Review by cescgina — CHANGES_REQUESTED** [review]
LGTM, just left a comment

File: cibyl/models/ci/system.py
Comment (bregman-arie) [reply cibyl/models/ci/system.py:50]: True :)

**Pull request merged**
